README.md

```diff
@@ -17,6 +17,6 @@ and merging branches to master you leave a lot of unused branches in your reposi
 
 Example:
 
-``` python
+``` bash
```

Comment:
Im thinking it's a bash command since we're not running it inside the py interpreter